test/unit/base.test.js

```diff
@@ -37,6 +38,25 @@ describe('CONVERT FUNCTION TESTS ', function() {
         expect(conversionResult.output[0].data).to.have.property('info');
         expect(conversionResult.output[0].data).to.have.property('item');
 
+        done();
+      });
+    });
+    it('[Github #90] - Should create a request using local server instead of global server ' +
```

Comment:
@Dhroov7 I see `Github #90` here and `Github: 102` somewhere else. Let's stick to one style (#) - that's what's generally used across other projects too.

Reply:
I will change the style in #103